trustpoint/templates/devices/details/devices_table.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
```

Comment:
I think it is a good idea to separate the table to a new sub-table, but I would put the file a layer higher, not into the `details` folder, as it is not related to the device details view.